tests/test_id.cpp

```diff
@@ -175,10 +175,8 @@ TEST(ID, WriteDataSignExtend)
 	IFID ifid;
 	Controls ctrl;
 
-	const std::array<uint16_t (*)(const uint8_t, const uint8_t, const uint8_t), 5> itypes = {
+	const std::array<uint16_t (*)(const uint8_t, const uint8_t, const uint8_t), 3> itypes = {
 		inst::addi,
-		inst::blt,
-		inst::beq,
```

Comment:
@aharris739 is it correct to remove these here?  I don't think that `write_data` should contain any meaningful data now that the ID stage handles the `new_PC_address` calculation.